common/lib/xmodule/xmodule/js/src/problem/edit.coffee

```diff
@@ -552,20 +537,22 @@ class @MarkdownEditingDescriptor extends XModule.Descriptor
       });
 
       // split scripts and preformatted sections, and wrap paragraphs
-      splits = xml.split(/(\<\/?(?:script|pre).*?\>)/g);
-      scriptFlag = false;
+      splits = xml.split(/(\<\/?(?:script|pre|label|description).*?\>)/g);
+
+      // used to check if a text string be wrapped by <p> tag  
```

Comment:
Typo in this comment.

And I'm still unsure what the conditions are in which a text string can/should be wrapped by a p tag. In words, when should wrappable be true vs. false?